src/frontends/tensorflow/include/openvino/frontend/tensorflow/node_context.hpp

```diff
@@ -21,6 +22,15 @@ class NodeContext : public ov::frontend::NodeContext {
     NodeContext(const std::shared_ptr<DecoderBase>& decoder, const OutputVector& inputs)
         : ov::frontend::NodeContext(decoder->get_op_type()),
           m_decoder(decoder),
+          m_translate_session(nullptr),
+          m_inputs(inputs) {}
+
+    NodeContext(TranslateSession* translate_session,
```

Comment:
Instead of a new ctor you could make session as last parameter and `nullptr` by default.

Reply:
Done.